arduino/WandCore/WandCore.ino

```diff
@@ -212,110 +213,75 @@ void setup()
   digitalWrite(LED_BUILTIN, LOW);
   setRGBLED(255, 255, 255); // white for power mode, which is default
 
+  // DEBUG ONLY
   // testDevices();
 }
 
-//void testDevices()
-//{
-//  // Test RGB LED
-//  setRGBLED(255, 0, 0);
-//  delay(1000);
-//  setRGBLED(0, 255, 0);
-//  delay(1000);
-//  setRGBLED(0, 0, 255);
-//  delay(1000);
-//  setRGBLED(0, 0, 0);
-//  delay(1000);
-//
-//  // Test color sensor
-//  setColorSensorLED(true);
-//  delay(1000);
-//  setColorSensorLED(false);
-//  for (int i = 0; i < 5; i++)
-//  {
-//    getRGB(&sensorR, &sensorG, &sensorB);
-//    delay(200);
-//  }
-//}
+/*
+ * Exercises attached devices 
+ */
+void testDevices()
+{
+  // Test RGB LED
+  setRGBLED(255, 0, 0);
+  delay(1000);
+  setRGBLED(0, 255, 0);
+  delay(1000);
+  setRGBLED(0, 0, 255);
+  delay(1000);
+  setRGBLED(0, 0, 0);
+  delay(1000);
+
+  // Test color sensor
+  setColorSensorLED(true);
+  delay(1000);
+  setColorSensorLED(false);
+  for (int i = 0; i < 5; i++)
+  {

```

Comment:
I think I did this, but if you want, I'd put this `{` on the same line as the `for`.